homeassistant/components/vera/manifest.json

```diff
@@ -1,9 +1,9 @@
 {
-  "domain": "vera",
-  "name": "Vera",
-  "config_flow": true,
-  "documentation": "https://www.home-assistant.io/integrations/vera",
-  "requirements": ["pyvera==0.3.13"],
-  "codeowners": ["@pavoni"],
-  "iot_class": "local_polling"
+    "domain": "vera",
```

Comment:
I think your editor has added tabs or something similar. Best to put this back as it was.